tests/syntax_tests/data/printer/expr/expected/newtype.res.txt

```diff
@@ -1,14 +1,12 @@
 let f = (type t, xs: list<t>) => ()
 let f = @attr (type t, xs: list<t>) => ()
-let f = (type t, xs: list<t>) => (type s, ys: list<s>) => ()
-let f = @attr (type t, xs: list<t>) => @attr2 (type s, ys: list<s>) => ()
+let f = (type t s, xs: list<t>, ys: list<s>) => ()
```

Comment:
This was incorrect before. It's a single function.
But if one toggles between types and terms, there's no way to know if trailing types belong to the current function or the next one. Moving all types first removes this ambiguity.